src/app/shared/components/template/components/dashed-box/dashed-box.component.ts

```diff
@@ -23,7 +24,7 @@ export class TmplDashedBoxComponent
 
   ngOnInit() {
     this.getParams();
-    this.innerHTML = this.domSanitizer.bypassSecurityTrustHtml(this._row.value);
+    this.innerHTML = this.domSanitizer.bypassSecurityTrustHtml(marked(this._row.value.toString()));
```

Comment:
Looks good, this seems like a neat way to do it. I've noticed that in the text component, we are applying a `markdown` pipe. It might make some sense to use that pipe here, to avoid this additional import of the "marked" library. Although the text component doesn't use `bypassSecurityTrustHtml()`, and I'm not sure they would be compatible. I think this is probably fine as it is.